nerfstudio/scripts/process_data.py

```diff
@@ -49,6 +49,9 @@ class ProcessRecord3D(BaseConverterToNerfstudioDataset):
     2. Converts Record3D poses into the nerfstudio format.
     """
 
+    ply: Optional[Path] = None
```

Comment:
Nit: Might want to call this `ply_dir` instead to emphasize that it's a directory, not a ply file itself.

Is the reason for doing a dir instead of the ply filepath because Record3D can generate multiple ply's?

Reply:
yea exactly, when you export Record3D gives a bunch of ply's, ill make the change rn